docs/Developers/Newcomers_Guide.rst

```diff
@@ -158,7 +158,7 @@ fork repository.
    will work.
 
    ::
-        $ git checkout -b <branchname>
+         $ git checkout -b <branchname>
```

Comment:
Sorry this did not fix the issue. Please see the we preview in git to check if you actually fix the issue.
it still renders like this:
````
::
    $ git checkout -b <branchname> 
````